main.c

```diff
@@ -4,53 +4,21 @@
 #include "groupTestIncludes.h"
 
 int main(int argc, char** argv) {
-    fileName = malloc(sizeof(char) * 10);
-    strcpy(fileName, "shape.txt");
-
     // Run tests if need be
     if (argc == 2 && strcmp(argv[1], "-t") == 0) {
         runAllTests();
-    } else if (argc == 2) {
-    //else if (argc == 3) {
-        //fileName = malloc(sizeof(char) * BUFSIZ);
-        //strncpy(fileName, argv[1], BUFSIZ);
+    } else if (argc == 4) {
+        inputFileName = malloc(sizeof(char) * BUFSIZ);
+        strncpy(inputFileName, argv[1], BUFSIZ);
+
+        transformationFileName = malloc(sizeof(char) * BUFSIZ); 
+        strncpy(transformationFileName, argv[2], BUFSIZ);
         
         outputFileName = malloc(sizeof(char) * BUFSIZ);
-        strncpy(outputFileName, argv[1], BUFSIZ);
-
-        //~~~~~ Group 1 ~~~~~//
-        inputShape = readInput(fileName);
-
-        // Instantiate Temporary Points //
-        // FIXME: REMOVE AFTER GROUP 1 IMPLEMENTATION //
-        inputShape->points = malloc(sizeof(struct point*) * 1);
-        inputShape->numOfPoints = 1;
-
-        struct point* temp = malloc(sizeof(struct point));
+        strncpy(outputFileName, argv[3], BUFSIZ);
         
-        temp->element[0] = 1.0f;
-        temp->element[1] = 1.0f;
-        temp->element[2] = 1.0f;
-        temp->element[3] = 1.0f;
-
-        setPoint(0, temp);
-
-        //GROUP 1 WILL IMPLEMENT THIS
-        setGlobalScale(1);
-        setXScale(1);
-        setYScale(1);
-        setZScale(1);
-        setXRotation(0);
-        setYRotation(0);
-        setZRotation(0);
-        setXTranslation(0);
-        setYTranslation(0);
-        setZTranslation(0);
-        setXShear(0);
-        setYShear(0);
-        setZShear(0);
-
```

Comment:
I know it isn't here but you need to include your tests at the bottom if you want to submit them (if not still add them but comment them out so we know you have them